{% extends '/base.html' %}

{% block headContent %}
    <title>Document | TCPGdb</title>


<script> // clicked button change color and revert//
var prevDiv=null
function changecolor(){ 
  if(prevDiv) {
    prevDiv.style.backgroundColor = "white";
  }
  var target = event.currentTarget
  target.style.backgroundColor="#d1d3d5" 
  selected = true                               
  prevDiv = target;
}
</script>

<!--scroll to section-->
<script> function go2sec1() { const element = document.getElementById("update");element.scrollIntoView();}</script>
<script> function go2sec2() { const element = document.getElementById("process");element.scrollIntoView();}</script>
<script> function go2sec3() { const element = document.getElementById("term");element.scrollIntoView();} </script>

{% endblock %}

{% block navContent %}
<li><a href="{{ url_for('datasets') }}"><strong>Datasets Summary</strong></a></li>
<li><a href="{{ url_for('crispr') }}"><strong>T-Screen</strong></a></li>
<li><a href="{{ url_for('score') }}"><strong>TPS-score</strong></a></li>
<li><a href="{{ url_for('search') }}"><strong>T-GEXP</strong></a></li>
<li><a href="{{ url_for('document') }}"><strong><span style="color: #ffffff;">Document</span></strong></a></li>
<li><a href="{{ url_for('contact') }}"><strong>Contact</strong></a></li>
{% endblock %}


{% block content %}

<div class="container-fluid ng-scope" style="text-align:justify; text-justify:inter-ideograph;">
    <div class="row">
        <div class="col-md-2">
            <div style="position:fixed;text-align:left;">
                <h4 class="text-purple"><span class="glyphicon glyphicon-list list-group"></span> Document</h4>
                <h4><a onclick="go2sec1();changecolor(this);" class="cursor-hand list-group-item">Update notes</a></h5>
                <h4><a onclick="go2sec2();changecolor(this);" class="cursor-hand list-group-item">Data process</a></h5>
                <h4><a onclick="go2sec3();changecolor(this);" class="cursor-hand list-group-item">Term of usage</a></h5>
            </div>
        </div>
        <div class="col-md-10">

            <div class="panel panel-default" id="update">
                <div class="panel-heading">
                    <h4>Update notes</h4>
                </div>
                <div class="panel-body ">
                   <div style="margin-right: 40px; margin-left: 12px;">
                        <p style="font-size:18px;font-family:Times, serif;">
                            <strong>&#9633&nbsp;2025 May&nbsp;</strong>
                            New CAR-T datasets and T cell screens added, see in Datasets Summary section! We added Shiny App for visualizing CAR-T single-cell RNAseq data, T cell subtypes were annotated using ScType. Feel free to explore!
                        </p>
                        <p style="font-size:18px;font-family:Times, serif;">
                            <strong>&#9633&nbsp;2024 Dec&nbsp;</strong>
                            The first edition of the TCPGdb website is now online!
                        </p>
                    </div>
                </div>
            </div>

            <div class="panel panel-default" id="process">
                <div class="panel-heading">
                    <h4>Data collection and process</h4>
                </div>
                <div class="panel-body ">
                    <div>
                        <span class="glyphicon glyphicon-bell" style="line-height : 20px ;color: #b52e31;font-size:20px;"> </span><strong style="line-height : 20px ;font-size:20px;color: #b52e31;font-family: Times, serif;"> Data collection</strong>
                        <br>
                 <section>
                    <div style="margin-right: 40px; margin-left: 12px;">
                        <p style="line-height : 30px ;font-size:18px;font-family:Times, serif;">&#9633 &nbsp;T cells screen datasets were obtained from Gene Expression Omnibus (GEO), and previously published literature through searching Pubmed and Google scholar with keywords “CRISPR screen AND T cell”. Both RNA-seq and microarray studies for featuring gene function in T/lymphocytes were retrieved from GEO and SRA as well.</p>
                    </div>
                </section>
                 <section>
                    <div style="margin-right: 40px; margin-left: 12px;">
                        <p style="line-height : 30px ;font-size:18px;font-family:Times, serif;">&#9633 &nbsp;For CRISPR/ORF screen datasets, we retrieved the raw sgRNA data from all screens and then processed each dataset using MAGeCK with uniform parameters. For datasets without raw data, the summarized data from original literature will be used.</p>
                    </div>
                </section>
                 <section>
                    <div style="margin-right: 40px; margin-left: 12px;">
                        <p style="line-height : 30px ;font-size:18px;font-family:Times, serif;">&#9633 &nbsp;For RNA-Seq data, SRA Toolkit was used to download the fastq files. Fastp  were employed for adapter sequence removal and  trimming to obtain high-quality clean reads. STAR aligner was used to align and quantify the reads in clean fastq files.</p>
                    </div>
                </section>
                 <section>
                    <div style="margin-right: 40px; margin-left: 12px;">
                        <p style="line-height : 30px ;font-size:18px;font-family:Times, serif;">&#9633 &nbsp;Processed data of single cell RNA-Seq data for CAR T cells were obtained from GEO.</p>
                    </div>
                </section>
                 <section>
                    <div style="margin-right: 40px; margin-left: 12px;">
                        <p style="line-height : 30px ;font-size:18px;font-family:Times, serif;">&#9633 &nbsp;R-package affy and oligo was used to process the raw microarray data.</p>
                    </div>
                </section>
                <p style="line-height : 30px ;font-size:18px;font-family:Times, serif;">All datasets accessions are listed in the Datasets Summary section.</p>
                    </div>

                    <div>
                        <span class="glyphicon glyphicon-bell" style="line-height : 20px ;color: #b52e31;font-size:20px;"> </span><strong style="line-height : 20px ;font-size:20px;color: #b52e31;font-family: Times, serif;"> Data analysis</strong>
                        <br>

                 <section>
                    <div style="margin-right: 40px; margin-left: 12px;">
                         <strong style="font-size:18px;color: #335238;font-family:Times, serif;">&#9633 &nbsp;CRISPR screen analysis using MAGeCK</strong>
                        <p style="line-height : 30px ;font-size:18px;font-family:Times, serif;">
                        All CRISPR screens were processed using MAGeCK with uniform parameters 'mageck test --remove-zero-threshold 0 --norm-method 'median' --adjust-method fdr'. For screens using mouse models, mouse genes were matched to their orthologous human genes using the biomaRt package in R.</p>
                    </div>
                </section>
                 <section>
                   <div style="margin-right: 40px; margin-left: 12px;">
                         <strong style="font-size:18px;color: #335238;font-family:Times, serif;">&#9633 &nbsp;Gene Set Enrichment Analysis</strong>
                        <p style="line-height : 30px ;font-size:18px;font-family:Times, serif;">
                        GSEA was performed to evaluate the effect of CRISPR screen gene targets at the genepathway level. The analyses were conducted using the R package clusterProfiler with curated gene set of <a href="http://geneontology.org/docs/go-enrichment-analysis/" target="_blank">biological process(BP)</a> category in gene ontology.</p>
                    </div>
                </section>
                <section>
                   <div style="margin-right: 40px; margin-left: 12px;">
                         <strong style="font-size:18px;color: #335238;font-family:Times, serif;">&#9633 &nbsp;Survival Analysis</strong>
                        <p style="line-height : 30px ;font-size:18px;font-family:Times, serif;">
                        The survival analysis of a specified gene in T cell lymphoma/leukemia dataset was calculated using a Kaplan–Meier (KM) model through R survival packages. P values less than 0.05 were considered statistically significant.</p>
                    </div>
                </section>
                <section>
                   <div style="margin-right: 40px; margin-left: 12px;">
                         <strong style="font-size:18px;color: #335238;font-family:Times, serif;">&#9633 &nbsp;T cell perturbation scoring</strong>
                        <p style="line-height : 30px ;font-size:18px;font-family:Times, serif;">
                        To address the bias in gene selection results from various T cell screening datasets, we introduced a T cell Perturbation Score (TPS). This score systematically measures the impact of specific genes on T cell function following CRISPR activation or interference by aggregating data from multiple screening studies. For detailed methodology, please refer to our manuscripts.</p>
                    </div>
                </section>
                </div>
            </div>
       </div>

            <div class="panel panel-default" id="term">
                <div class="panel-heading">
                    <h4>Term of usage</h4>
                </div>
                <div class="panel-body ">
                    <div style="margin-right: 50px; margin-left: 15px;">
                        <p style="vertical-align: top;display: inline-block;line-height : 30px ;font-size:18px;font-family:Times New Roman, Times, serif;">
                        TCPGdb is a database providing comprehensive expression resources and functional analysis of T cell gene functionalities by Chen lab. Our team makes no warranties or representations, express or implied, with respect to any of the Content, including as to the present accuracy, completeness, timeliness, adequacy, or usefulness of any of the Content. By using this website, you agree that Chen lab will not be liable for any losses or damages arising from your use of or reliance on the Content, or other websites or information to which this website may be linked.<br>
                        </p>
                        <p style="vertical-align: top;display: inline-block;line-height : 30px ;font-size:18px;font-family:Times New Roman, Times, serif;">
                        You may not copy, transfer, reproduce, modify or create derivative works of TCPGdb for any commercial purpose without the express permission of Chen lab.
                            If you seek to use TCPGdb for such purposes, please request the license which best describes your anticipated use of TCPGdb below:
                        <br>
                        1.  Research use in commercial setting
                        <br>
                        2.  Use in a commercial product
                        <br>
                        3.  Use for patient services or reports in a hospital setting
                        <br><br>
                        For additional information, please contact Dr. Chen at sidi.chen@yale.edu.
                        <br>
                        </p>

                    </div>
                </div>
            </div>

    </div>
</div>
</div>

{% endblock content %}